spec/controllers/sessions_controller_spec.rb

```diff
@@ -60,111 +60,6 @@
     end
   end
 
-  describe "GET new" do
-    it "works when logged out" do
-      get :new
-      expect(response.status).to eq(200)
-      expect(assigns(:page_title)).to eq("Sign In")
-    end
-
-    it "redirects when logged in" do
-      login
-      get :new
-      expect(response).to redirect_to(continuities_url)
-      expect(flash[:error]).to eq("You are already logged in.")
-    end
-  end
-
-  describe "POST create" do
```

Comment:
We, uh, probably do want to test this works? Logging out, too.

Reply:
This was moved to `spec/requests/user_spec.rb` since it's no longer part of our usual sessions controller (I think Red committed that already, not sure, tests are WIP)

Reply:
Ah, I wasn't sure how much those were testing the same behavior as these tests covered, fair enough.